docs/excel/custom-functions-overview.md

```diff
@@ -373,7 +373,7 @@ function getComment(x) {
 - **May 7, 2018**: Shipped* support for Mac, Excel Online, and synchronous functions running in-process
 - **September 20, 2018**: Shipped support for custom functions JavaScript runtime. For more information, see [Runtime for Excel custom functions](custom-functions-runtime.md).
 
-\* to the Office Insiders Channel
+\* to the [Office Insider](https://products.office.com/office-insider) channel (formerly called "Insider Fast")
```

Comment:
The terms channel and level are both used to describe Office Insider - Insider. Are they interchangeable?

Reply:
Thanks for the feedback @AlexJerabek and nice attention to detail :) Yes, I'd say those terms are interchangeable.  When you join the Office Insider program, you have a choice of two "levels" -- **Insider** and **Monthly channel**.  Until recently, the first "level" was called the "Insider Fast" _channel_ and that's still the term that's most widely recognized.